editor/test/resources/save_data_project/checked.model

```diff
@@ -1,7 +1,25 @@
-mesh: "/builtins/assets/meshes/cube.dae"
-material: "/builtins/materials/model.material"
-textures: "/referenced/images/white.png"
-skeleton: "/builtins/assets/meshes/cube.dae"
-animations: "/builtins/assets/meshes/cube.dae"
-default_animation: "cube"
+mesh: "/referenced/mesh.glb"
+skeleton: "/referenced/mesh.glb"
+animations: "/referenced/mesh.glb"
+default_animation: "cube_flip"
 name: "checked"
+materials {
+  name: "material0"
+  material: "/builtins/materials/model.material"
+  textures {
+    sampler: "tex0"
+    texture: "/referenced/images/red.png"
+  }
+}
+materials {
+  name: "material1"
+  material: "/referenced/model.material"
+  textures {
+    sampler: "texture0"
+    texture: "/referenced/images/green.png"
+  }
+  textures {
+    sampler: "texture1"
+    texture: "/referenced/images/blue.png"
+  }
+}
```

Comment:
The `checked.model` has been updated to the new file format now that we have multiple material and texture assignments. But I added `legacy_material_and_textures.model` with the old file format to the `silent_migrations` folder for use with the `silent-migrations-test`.